csv/shared/src/main/scala/fs2/data/csv/CellDecoder.scala

```diff
@@ -191,10 +191,10 @@ object CellDecoder extends CellDecoderInstances1 with LiteralCellDecoders with E
       case d                  => new DecoderError(s"parsed duration isn't finite: $d").asLeft
     }
 
-  implicit val javaUrlDecoder: CellDecoder[URL] = s =>
+  implicit val javaUrlDecoder: CellDecoder[URI] = s =>
```

Comment:
I wouldn't remove the `URL` one, since it's part of the API. Maybe just add the `URI` variation and make the `URL` one use it.

Reply:
Is it ok to do `CellDecoder/Encoder[URL]` only for the jvm side? otherwise the tests won't link on the scala.js side?

Reply:
Sure, this is fine.